crates/core/tedge_actors/src/message_boxes.rs

```diff
@@ -142,44 +143,46 @@ pub trait MessageBox: 'static + Sized + Send + Sync {
     fn logging_is_on(&self) -> bool;
 }
 
+enum WrappedInput<Input> {
+    Message(Input),
+    RuntimeRequest(RuntimeRequest),
+}
+
+// TODO: rename
+#[async_trait]
+trait Foo<Input> {
```

Comment:
Not sure this is the right set of fn's

Reply:
I took these names, reworded/expanded the docs - I picked a name happy to change it.